vc-verifier/js/src/CredentialsVerifier.js

```diff
@@ -40,22 +41,38 @@ const verifyCredentialSignature = (jwsHeaderAlgoName, publicKey, actualData, sig
     else if (jwsHeaderAlgoName === Constants.JWS_RS256_SIGN_ALGO_CONST) {
         isVerified = crypto.verify('RSA-SHA256', actualData, publicKey, signature);
     }
-    else {
-        throw new Error('Verification algorithm not supported');
-    }
     return isVerified;
 }
 
 export const verifyCredentials = async (credential) => {
```

Comment:
We should keep kotlin and js interface same.
So it should be verify which accepts credential and format

Reply:
Updated verify with credential and format params and annotated verifyCredentials as deprecated.

Reply:
Since Factory pattern is not introduced and js library supports only ldp format with RSA proof type VCs for now, format param will be unused until Factory pattern is added.